src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/pages/AttributeGroupsIndex.tsx

```diff
@@ -33,9 +33,13 @@ const AttributeGroupsIndex: FC = () => {
     useSelection<AttributeGroup>(attributeGroups.length);
   const translate = useTranslate();
   const settingsHomePageRoute = `#${useRoute('pim_settings_index')}`;
-
   const [groupCount, setGroupCount] = useState<number>(attributeGroups.length);
 
+  const [childrenAttributesCount, targetAttributeGroups] = getImpactedAndTargetAttributeGroups(
```

Comment:
Maybe more available target attribute groups here no ?
```suggestion
  const [childrenAttributesCount, availableTargetAttributeGroups] = getImpactedAndTargetAttributeGroups(
```